linode/manager
# M3-5913: Replace `novnc-node` with `react-vnc`

## Description 📝 

- `novnc-node` was last updated 7 years ago so we are replacing it with a more modern and supported alternative
- `react-vnc` will play better with Vite when we make the switch
- Cleans up a lot of old code and simplifies logic

## How to test 🧪

- Test **Lish** and **Glish** functionality on Linodes
  - Check that it is usable for a long time because tokens expire quickly and Cloud should handle reconnecting when a new token is needed
  - Test with Powered Off Linode
  - Compare to Production and check for any regressions in functionality
